simulator/src/css/main.stylesheet.css

```diff
@@ -564,6 +564,8 @@ div.icon img {
   -khtml-user-drag: none;
   -moz-user-drag: none;
   -o-user-drag: none;
+  pointer-events: none;
+
```

Comment:
Can you please remove this extra line?

Reply:
Done but was it really that necessary? 😅
I mean does it reflect on the website's performance or something that I don't know about?

Reply:
Even O(n^3) complexity run and O(n) run the difference in quality, I have learned quality code in CircuitVerse so would always look for quality whether is reflected or not!

Reply:
Yep absolutely. We usually have powerful enough hardware to run slightly more unoptimized code, but a developer looking at slightly more messy code can really screw their productivity. Although just an extra newline is not that big of a deal, but code linting overall is pretty important